File: imobil-project/bin/Proprietar.dart
import 'Persoana.dart';
class Proprietar extends Persoana
{
  int pretcerut;
  Proprietar(String nume, int varsta, int telefon, String email, int pretcerut) : super(nume, varsta, telefon, email)
  {
    this.pretcerut = pretcerut;
  }
  int get Pret_Cerut => pretcerut;
  set Pret_Cerut(int value) => pretcerut=value;
  @override
  String toString() {
    StringBuffer sb = new StringBuffer();
    sb.writeln('*');
    sb.writeln('NUME: ' + nume + ', ');
    sb.writeln('VARSTA: ' + varsta.toString() + ', ');
    sb.writeln('NUMAR DE TELEFON: ' + telefon.toString() + ', ');
    sb.writeln('EMAIL: ' + email + ', ');
    sb.writeln('PRET CERUT: $pretcerut');
    sb.writeln('* ');
    return sb.toString();
  }
  }

File: Imobil C#/Imobil/Proprietar.cs
using System.Text;

namespace Imobil
{
    public class Proprietar: Persoana
    {
        public int pretcerut;
        public Proprietar(string nume, int varsta, long telefon, string email, int pretcerut) : base(nume, varsta, telefon, email)
        {
            this.pretcerut = pretcerut;
        }

        public int PretCerut
        {
            get
            {
                return pretcerut;
            }
            set
            {
                pretcerut = value;
            }
        }
        public override string ToString()
        {
            StringBuilder sb = new StringBuilder();
            sb.Append("*");
            sb.Append("NUME: " + nume + ", ");
            sb.Append("VARSTA: " + varsta + ", ");
            sb.Append("NUMAR DE TELEFON: " + telefon + ", ");
            sb.Append("EMAIL: " + email+", ");
            sb.Append("PRET CERUT: " + pretcerut);
            sb.Append("*");
            return sb.ToString();
        }
    }
}